sql/distinct.go

```diff
@@ -37,6 +37,9 @@ func (*planner) distinct(n *parser.Select, p planNode) planNode {
 		d.columnsInOrder = make([]bool, len(p.Columns()))
 	}
 	for _, p := range p.Ordering() {
+		if p == 0 {
```

Comment:
`break` matches up with the previous behavior which would have cut the ordering slice off when an index of 0 would have been returned.

I think we should sit down next week and talk about the current implementation and where the biggest warts and complexities are.